### PR TITLE
UX: adds visual feedback to topic-notifications-button

### PR DESCRIPTION
**Pull request by jjaffeux**
Screenshot 2020-02-07 at 14 48 13

### CODE REVIEW

File: app/assets/javascripts/select-kit/components/topic-notifications-button.js.es6 [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
+  }),
 
   actions: {
     changeTopicNotificationLevel(newNotificationLevel) {
       if (newNotificationLevel !== this.notificationLevel) {
-        this.topic.details.updateNotifications(newNotificationLevel);
+        this.set("isLoading", true);
+        this.topic.details
+          .updateNotifications(newNotificationLevel)
+          .finally(() => later(() => this.set("isLoading", false), 250));
```
Comment (eviltrout) [inline app/assets/javascripts/select-kit/components/topic-notifications-button.js.es6]: Why do we need the 250ms delay here?

Comment (jjaffeux) [reply app/assets/javascripts/select-kit/components/topic-notifications-button.js.es6]: Sorry forgot to answer, this is to have a consistent loading animation. Often the loading is so fast that you barely see it and end up just having a blinking/clunky UI. Given this is only for animation purpose and doesn't block UI I think it's fine to always have this delay.

Comment (eviltrout) [reply app/assets/javascripts/select-kit/components/topic-notifications-button.js.es6]: Sounds good to me. Thanks for the clarification.

**Comment by SamSaffron** [comment]
@jjaffeux  looks like we have conflict :( can you fix and merge this, change looks pretty safe to me.

**Comment by jjaffeux** [comment]
@SamSaffron  I want to try a better a solution, this is why I postponed merged for now.

**Comment by SamSaffron** [comment]
I worry that we are keeping this open for too long @jjaffeux  closing for now pending a new PR

**Pull request closed without merging**